Sources/MapboxNavigation/RouteLineController.swift

```diff
@@ -149,18 +149,14 @@ extension NavigationMapView {
         // MARK: - NavigationComponentDelegate implementation
         
         func navigationViewDidLoad(_ view: UIView) {
-            navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { [weak self] _ in
+            navigationMapView.mapView.mapboxMap.onEvery(.styleLoaded) { [weak self] _ in
                 self?.navigationMapView.localizeLabels()
                 self?.navigationMapView.mapView.showsTraffic = false
+                self?.showRouteIfNeeded()
                 
                 // FIXME: In case when building highlighting feature is enabled due to style changes and no info currently being stored
                 // regarding building identification such highlighted building will disappear.
             }
-            
-            // Route line should be added to `MapView`, when its style changes.
-            navigationMapView.mapView.mapboxMap.onEvery(.styleLoaded) { [weak self] _ in
-                self?.showRouteIfNeeded()
```

Comment:
I consolidated this operation with the block above. I’m pretty sure all of these style customizations need to be reapplied every time the style changes.